client/scss/components/_panel.scss

```diff
@@ -0,0 +1,89 @@
+$header-icon-size-sm: theme('spacing.5');
+$header-button-size-sm: theme('spacing.8');
+
+.w-panel {
+  margin-bottom: theme('spacing.10');
+  // Legacy code support. Make sure we render correctly enough after a panel that contains un-cleared floats.
+  clear: both;
+}
+
+.w-panel__header {
+  display: flex;
+  align-items: center;
+  margin-bottom: theme('spacing.4');
+  margin-inline-start: calc(-1 * theme('spacing.5'));
+
+  @include media-breakpoint-up(sm) {
+    $gap: theme('spacing.1');
+    gap: $gap;
+    margin-inline-start: calc(-1 * 2 * ($header-button-size-sm + $gap));
+  }
+}
+
+.w-panel__heading {
+  @apply w-h3;
+  display: inline-block;
+  margin: 0;
+}
+
+.w-panel__heading--label {
+  @apply w-label-1;
+}
+
+.w-panel__anchor,
+.w-panel__toggle {
+  display: inline-grid;
+  justify-content: center;
+  align-content: center;
+  color: theme('colors.primary.DEFAULT');
+  padding: 0;
+
+  @include media-breakpoint-up(sm) {
+    width: $header-button-size-sm;
+    height: $header-button-size-sm;
+  }
+}
+
+.w-panel__anchor {
+  // Only hide anchors for devices that support hover interactions.
+  @media (hover: hover) {
+    // Hiding with opacity only, so the anchor isn’t skipped in the page’s focus order.
+    opacity: 0;
+
+    .w-panel__header:hover &,
+    .w-panel__header:focus-within & {
+      opacity: 1;
+    }
+  }
+}
+
+.w-panel__toggle {
+  appearance: none;
+  background: transparent;
+}
+
+.w-panel__icon {
+  width: theme('spacing.3');
+  height: theme('spacing.3');
+
+  @include media-breakpoint-up(sm) {
+    width: $header-icon-size-sm;
+    height: $header-icon-size-sm;
+  }
+
+  // Only rotate the default caret icon, not custom ones.
+  [aria-expanded='false'] &.icon-caret-down {
+    transform: rotate(-90deg);
```

Comment:
will this animate? should this animate?

Reply:
I tested locally and saw it does not animate and do not think it's needed. Maybe a nice to have but the UI is clear as is.